LLama.SemanticKernel/ExtensionMethods.cs

```diff
@@ -3,9 +3,9 @@
 
 namespace LLamaSharp.SemanticKernel;
 
-internal static class ExtensionMethods
+public static class ExtensionMethods
 {
-    internal static global::LLama.Common.ChatHistory ToLLamaSharpChatHistory(this ChatHistory chatHistory)
+    public static global::LLama.Common.ChatHistory ToLLamaSharpChatHistory(this ChatHistory chatHistory)
```

Comment:
I think adding a parameter `bool ignoreCase = true` to allow control on it will be better.

Reply:
Makes sense to me, just added the parameter and thank you for the awesome library! This is actually my first contribution to an open source project :smile:

Reply:
You're always welcome! I'm happy that our work makes sense to you. If you're interested in developing this library, there's a TODO list in [LLamaSharp project](https://github.com/orgs/SciSharp/projects/5). 😊

Reply:
Sure, I'd love to help! It looks like the project board is set to private though, I'm getting a 404 error

Reply:
Sorry for that, I've now changed it to public. :)